### PR TITLE
Clarify constructor arguments.

### PR DESCRIPTION
**Pull request by chriseth**
Fixes https://github.com/ethereum/solidity/issues/4394

### CODE REVIEW

File: docs/miscellaneous.rst [hunk 1]
```diff
@@ -87,10 +87,14 @@ Solidity always places new objects at the free memory pointer and memory is neve
 Layout of Call Data
 *******************
 
-When a Solidity contract is deployed and when it is called from an
-account, the input data is assumed to be in the format in :ref:`the ABI
-specification <ABI>`. The ABI specification requires arguments to be padded to multiples of 32
-bytes.  The internal function calls use a different convention.
+The input data for a function call is assumed to be in the format in the :ref:`ABI
```
Comment (leonardoalt) [inline docs/miscellaneous.rst]: Would `...assumed to be in the :ref: ABI specification <ABI> format.` be better?

Comment (chriseth) [reply docs/miscellaneous.rst]: .. in the format defined by the ABI specification?

Comment (leonardoalt) [reply docs/miscellaneous.rst]: Sounds good!

**Comment by chriseth** [comment]
Updated.

**Review by leonardoalt — APPROVED** [review]

**Pull request merged**